python/ray/experimental/channel/nccl_group.py

```diff
@@ -103,6 +146,20 @@ def __init__(
                 cuda_stream, device_id=device.index
             )
 
+            if use_communication_streams:
+                import torch
+
+                self._send_stream = cp.cuda.ExternalStream(
+                    torch.cuda.Stream().cuda_stream, device_id=device.index
+                )
+                self._recv_stream = cp.cuda.ExternalStream(
+                    torch.cuda.Stream().cuda_stream, device_id=device.index
+                )
+                event = cp.cuda.Event()
+                event.record(cp.cuda.get_current_stream())
```

Comment:
A dumb CUDA question: what's the "current stream" here? Is it the CUDA stream for computation?

Reply:
current stream here is "compute stream"

Reply:
yes current stream is the current stream used for GPU operations unless otherwise specified